### ISSUE TITLE
Profiles' remotePath is ignored and sync to wrong remote path

### ISSUE THREAD

**Issue by jimlinntu** [liximomo/vscode-sftp#317]
## Expected Behavior
When I use "profiles" and set "remotePath" in it, it should sync to "remotePath"
## Actual Behavior
However, I found that it doesn't sync to "remotePath" but sync to my home directory
## Steps to Reproduce the Problem

  1. Set up configuration as: 
```
{
    "context": "./my_subdir",
    "protocol": "sftp",
    "syncMode": "update",
    "ignore": [
        ".vscode",
        ".git",
        ".DS_Store"
    ],
    "connectTimeout": 7200000,
    "profiles": {
        "cl6":{
            "host": "myhost_machine_url",
            "remotePath": "/nfs/home/[myAccount]/TEST2/",
            "username": myaccount"
        }
    }
}

```
  2. `SFTP: Upload Project`

### Extension Logs
  1.  Open User Settings.

      * On Windows/Linux - File > Preferences > Settings
      * On macOS - Code > Preferences > Settings
  2. Set `sftp.printDebugLog` to `true` and reload vscode.
  3. Reproduce the problem, get the logs from View > Output > sftp.
 
Logs:
```
[trace] run command 'Upload Project'
[trace] handle upload folder for [myLocalPath]
[trace] transfer from [myLocalPath] to ./
```

## Specifications

  - Version: 1.5.5
  - Platform: Windows 10

**Comment by simonit72** [comment]
Also have this issue since the latest release.
Files now end up in /home/username followed by the active file path.  Remote path is being ignored.

**Comment by liximomo** [comment]
Fix in v1.5.6.
